app/models/katello/subscription.rb

```diff
@@ -17,6 +17,10 @@ def self.with_subscribable_content
         where("#{Katello::Product.table_name}.id" => Product.with_subscribable_content)
     end
 
+    def self.virt_who
+      joins(:pools).where("#{Katello::Pool.table_name}.virt_who" => true)
```

Comment:
Technically this is a scope (http://guides.rubyonrails.org/active_record_querying.html#scopes) I can covert this to use the `scope` method if that would make it more clear.